### PR TITLE
Add `Kyoku 2024 Results` news post

### PR DESCRIPTION
**Pull request by 0x84f**
- [x] add assets.ppy.sh links to videos
[preview](https://osu3.roanh.dev/home/news/2024-09-19-kyoku-2024-results)

### CODE REVIEW

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-2nd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Zero Blunders for managing to lock in a total score of **86.00**, securing 2nd place and 1 contest point for all team members.
+
+### 1st place - cosmo/LOGIΛ
+
+Team cosmo/LOGIΛ have managed to concoct what can best be described as a Grand Finals Tiebreaker map on steroids. [Stellar Evolution](https://osu.ppy.sh/beatmapsets/2242303) from [airlemoneX](https://soundcloud.com/airlemonex) has hints of [Camellia](https://osu.ppy.sh/beatmaps/artists/31) as the inspiration behind the track, with expert mappers [Lulu-](https://osu.ppy.sh/users/4201715), [Slifer](https://osu.ppy.sh/users/15084122) and [Halgoh](https://osu.ppy.sh/users/4109923) fully playing into it with a focus on visuals that do not compromise gameplay.
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:39]:
```suggestion
Team cosmo/LOGIΛ managed to concoct what can best be described as a Grand Finals tiebreaker map on steroids. [Stellar Evolution](https://osu.ppy.sh/beatmapsets/2242303) from [airlemoneX](https://soundcloud.com/airlemonex) has hints of [Camellia](https://osu.ppy.sh/beatmaps/artists/31) as the inspiration behind the track, with expert mappers [Lulu-](https://osu.ppy.sh/users/4201715), [Slifer](https://osu.ppy.sh/users/15084122) and [Halgoh](https://osu.ppy.sh/users/4109923) fully playing into it with a focus on visuals that do not compromise gameplay.
```

Comment (Walavouchey) [reply news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:39]: > visuals that do not compromise gameplay

hmm sure but it's quite hard to play as a result of those visuals sometimes O.o

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+| 29 | 17.75 | Team boss baby | [![SoundCloud](/wiki/shared/link/SoundCloud.png)](https://soundcloud.com/resolve1k/evo "SoundCloud") | inno & tekea - evo |
+| 30 | 13.75 | nou |  | JMicc - Vicious Onslaught (Zuure) (2) |
+| 31 | 10.50 | Endlessly Small Bombs |  | treegas500 - Prelude To An Agitated Spring |
+| 32 | 4.00 | Spojený Sekoitus | [![SoundCloud](/wiki/shared/link/SoundCloud.png)](https://soundcloud.com/totalnormalfry/kyoku-2024stalled-superiority "SoundCloud") | TotalNormalFry & HappyAdam - Stalled Superiority (PuffballYT) |
+| 33 | 3.25 | :sob: |  | V3ARI - Light In The Dark |
+| *DNS* |  | Team Blue but Red | [![SoundCloud](/wiki/shared/link/SoundCloud.png)](https://soundcloud.com/redside100/ultraviolet "SoundCloud") [![YouTube](/wiki/shared/link/YouTube.png)](https://www.youtube.com/watch?v=FwgTUyMrxko "YouTube") | Redside - Ultraviolet |
+
+---
+
+Thank you to all the organizers, helpers, and participants for taking part of this one-of-a-kind event and creating a huge amount of original, high quality content for the community to enjoy.
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:96]:
```suggestion
Thank you to all the organisers, helpers, and participants for taking part of this one-of-a-kind event and creating a huge amount of original, high-quality content for the community to enjoy.
```

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+layout: post
+title: "Kyoku: Original Song & Beatmapping Contest - Results"
+date: 2024-09-19 19:30:00 +0000
+---
+
+Kyoku has concluded, so it's time to look at the final results, and check out all the amazing work by both musicians and mappers alike.
+
+![](https://assets.ppy.sh/contests/211/banner.jpg)
+
+If you don't recall, [Kyoku](/wiki/Contests/Kyoku/2024) was a contest where teams of mappers and music producers needed to work together and collaborate on creating both a brand-new original song, *and* a beatmap to go along with it. Inspired by [BMS OF FIGHTERS](https://www.bmsoffighters.net/), teams have spent the last **3 months** crafting their own works of art, all for a chance for become the top team and win both a prestigious badge, along with highly sought-after [contest points](/wiki/Contests/Contest_points).
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:11]: aside from "3 months" of mapping/composing it'd be good to mention that this contest started back in april (with a link to the announcement)

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-3rd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Team RGB as they scored themselves 3rd place with a final score of **84.50**!
+
+### 2nd place - Zero Blunders
+
+Team Zero Blunders has put their best foot forward in this contest with a genre mash-up song featuring many osu! references. [*who's afraid?*](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280) by fur:trash (also known as [Nihkee](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280)) offers many chances for [Mysty](https://osu.ppy.sh/users/10210657), [Le Mirai](https://osu.ppy.sh/users/13646997), [hakashii](https://osu.ppy.sh/users/11688893) & [[-Evil-]](https://osu.ppy.sh/users/10234313) to express their mapping ideas through music genres such as dariacore, dubstep & hyperpop, creating a balanced map that is both fun to look at and play.
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:29]:
```suggestion
Team Zero Blunders has put their best foot forward in this contest with a genre mash-up song featuring many osu! references. [*who's afraid?*](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280) by fur:trash (a.k.a. [Nihkee](https://osu.ppy.sh/users/14244740) on osu!) offers many chances for [Mysty](https://osu.ppy.sh/users/10210657), [Le Mirai](https://osu.ppy.sh/users/13646997), [hakashii](https://osu.ppy.sh/users/11688893) and [[-Evil-]](https://osu.ppy.sh/users/10234313) to express their mapping ideas via music genres such as dariacore, dubstep and hyperpop, creating a balanced map that is both fun to look at and play.
```

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-1st-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+With a final score of **88.25**, cosmo/LOGIΛ have managed to achieve first place, earning badges along with 2 contest points. Congratulations!
+
+![](/wiki/Contests/Kyoku/2024/img/badge.png)
+
+## Submissions
+
+It is highly recommended that you check out all the entries as every single team put blood, sweat and tears into their submissions. You can either check out the full VOD from the results reveal below, or check out and download individual submissions and give them a rating yourself.
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:51]: a different kind of dramatic

```suggestion
Every entry deserves your attention for the monumental amount of effort and late nights put into each and every song and map. You can either check out the full VOD from the results reveal below, or download the individual submissions and give them a rating yourself.
```

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Kyoku: Original Song & Beatmapping Contest - Results"
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:3]: subtitle could be omitted for brevity

```suggestion
title: "Kyoku 2024 Results"
```

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+date: 2024-09-19 19:30:00 +0000
+---
+
+Kyoku has concluded, so it's time to look at the final results, and check out all the amazing work by both musicians and mappers alike.
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:7]: maybe more like this. fewer commas and more flowy

```suggestion
While the seasons drifted over our heads, mappers and musicians were busy competing to create the greatest beatmaps of their own creation.
```

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+With a final score of **88.25**, cosmo/LOGIΛ have managed to achieve first place, earning badges along with 2 contest points. Congratulations!
+
+![](/wiki/Contests/Kyoku/2024/img/badge.png)
+
+## Submissions
+
+It is highly recommended that you check out all the entries as every single team put blood, sweat and tears into their submissions. You can either check out the full VOD from the results reveal below, or check out and download individual submissions and give them a rating yourself.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/kzsxu8254u4" frameborder="0" allowfullscreen></iframe>
```
Comment (Walavouchey) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:54]: the beginning includes info directed towards teams which isn't very relevant here ([mentioned in the thread](https://discord.com/channels/188630481301012481/1237198052054274058/1279934527074209913))

```suggestion
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/kzsxu8254u4?start=56" frameborder="0" allowfullscreen></iframe>
```

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md (hunk 3 above)
Comment (0x84f) [reply news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md]: https://github.com/ppy/osu-wiki/pull/12165/commits/23d01913b31d87fb86f160830ab3fd52a73fff98

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md (hunk 1 above)
Comment (0x84f) [reply news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md]: https://github.com/ppy/osu-wiki/pull/12165/commits/23d01913b31d87fb86f160830ab3fd52a73fff98

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+date: 2024-09-19 19:30:00 +0000
+---
+
+Kyoku has concluded, so it's time to look at the final results, and check out all the amazing work by both musicians and mappers alike.
+
+![](https://assets.ppy.sh/contests/211/banner.jpg)
+
+If you don't recall, [Kyoku](/wiki/Contests/Kyoku/2024) was a contest where teams of mappers and music producers needed to work together and collaborate on creating both a brand-new original song, *and* a beatmap to go along with it. Inspired by [BMS OF FIGHTERS](https://www.bmsoffighters.net/), teams have spent the last **3 months** crafting their own works of art, all for a chance for become the top team and win both a prestigious badge, along with highly sought-after [contest points](/wiki/Contests/Contest_points).
+
+With 76 total entries (resulting in 33 submitted entries) the competition was fierce, and the judges had a lot of work ahead of them. After much deliberation, the dust has settled, and we are proud to present the top 3 entries.
```
[the remark this answers is not in the document]
Comment (0x84f) [reply news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md]: https://github.com/ppy/osu-wiki/pull/12165/commits/23d01913b31d87fb86f160830ab3fd52a73fff98

**Review by peppy — DISMISSED** [review]
Haven't yet gone through the winners, but the post looks good from my end.

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-3rd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Team RGB as they scored themselves 3rd place with a final score of **84.50**!
+
+### 2nd place - Zero Blunders
+
+Team Zero Blunders has put their best foot forward in this contest with a genre mash-up song featuring many osu! references. [*who's afraid?*](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280) by fur:trash (a.k.a. [Nihkee](https://osu.ppy.sh/users/14244740) on osu!) offers many chances for [Mysty](https://osu.ppy.sh/users/10210657), [Le Mirai](https://osu.ppy.sh/users/13646997), [hakashii](https://osu.ppy.sh/users/11688893) and [[-Evil-]](https://osu.ppy.sh/users/10234313) to express their mapping ideas via music genres such as dariacore, dubstep and hyperpop, creating a balanced map that is both fun to look at and play.
```
Comment (Omekyu) [inline news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:29]: My name (Omekyu, https://osu.ppy.sh/users/14348073) is missing here, even though I was part of Team Zero Blunders as well. 
https://osu.ppy.sh/wiki/en/Contests/Kyoku/2024
image

**Review by Omekyu — CHANGES_REQUESTED** [review]

File: news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md (hunk 10 above)
Comment (0x84f) [reply news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md:29]: @Walavouchey are you gonna be issuing updates to the wiki page? ill amend the news post now

Comment (Walavouchey) [reply news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md]: they're already in the wiki page

File: wiki/Contests/Kyoku/2024/en.md [hunk 11]
```diff
@@ -17,12 +17,18 @@
 
 | Ranking | Prizes |
 | :-- | :-- |
-| Winning team | $100 USD + profile badge (pending) + [contest points](/wiki/Contests/Contest_points) |
+| Winning team | $100 USD + profile badge + 2 [contest points](/wiki/Contests/Contest_points) |
+| 2nd place | 1 contest point |
+
+![Kyoku 2024 winner's badge](img/badge.png)
```
Comment (Hiviexd) [inline wiki/Contests/Kyoku/2024/en.md:23]: why not use the assets.ppy.sh link for this one?

Comment (Walavouchey) [reply wiki/Contests/Kyoku/2024/en.md:23]: because most tournaments and contests don't do that

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**